test/src/e2e_vm_tests/test_programs/should_pass/language/enum_init_fn_call/test.ir

```diff
@@ -0,0 +1,55 @@
+script {
+    fn main() -> u64 {
+        local ptr { u64 } t
+
+        entry:
+        v0 = const { u64 } { u64 42 }, !1
+        br block3
+
+        block3:
+        v1 = phi(entry: v0)
+        v2 = const { u64, ( u64 | { u64 } ) } { u64 1, ( u64 | { u64 } ) undef }, !2
+        v3 = insert_value v2, { u64, ( u64 | { u64 } ) }, v1, 1, !2
+        v4 = extract_value v3, { u64, ( u64 | { u64 } ) }, 0, !3
+        v5 = const u64 1, !3
+        v6 = cmp eq v4 v5, !3
+        cbr v6, block0, block1, !3
+
+        block0:
+        v7 = extract_value v3, { u64, ( u64 | { u64 } ) }, 1, 1, !4
+        v8 = get_ptr ptr { u64 } t, ptr { u64 }, 0, !4
+        store v7, ptr v8, !4
+        v9 = get_ptr ptr { u64 } t, ptr { u64 }, 0, !5
+        v10 = extract_value v9, { u64 }, 0, !6
+        br block2
+
+        block1:
+        v11 = const u64 0, !7
+        br block2
+
+        block2:
+        v12 = phi(block0: v10, block1: v11)
+        ret u64 v12
+    }
+
+    fn anon_0() -> { u64 } {
+        entry:
+        v0 = const { u64 } { u64 undef }, !1
+        v1 = const u64 42, !8
+        v2 = insert_value v0, { u64 }, v1, 0, !1
+        ret { u64 } v2
+    }
+}
+
+!0 = filepath "enum_init_fn_call/src/main.sw"
+!1 = span !0 188 200
+!2 = span !0 63 95
+!3 = span !0 140 141
+!4 = span !0 132 133
+!5 = span !0 149 150
+!6 = span !0 50 52
+!7 = span !0 163 164
+!8 = span !0 196 198
+
+Compiling insert value
+  Bytecode size is 148 bytes.
```

Comment:
Does it pass with this still?!

Reply:
Actually, hang on, why is this file here?  I was thinking it was in the unit tests...

Reply:
You're right.. this just slipped through 😂

Reply:
Removed.